packages/thirdweb/src/react/web/ui/ConnectWallet/ConnectWallet.tsx

```diff
@@ -40,10 +45,11 @@ const TW_CONNECT_WALLET = "tw-connect-wallet";
  * Refer to [ConnectButtonProps](https://portal.thirdweb.com/references/typescript/v5/ConnectButtonProps) to see the available props.
  * @component
  */
-export function ConnectButton({ auth, ...props }: ConnectButtonProps) {
+export function ConnectButton(props: ConnectButtonProps) {
   const wallets = props.wallets || getDefaultWallets();
+  const activeAccount = useActiveAccount();
   const localeQuery = useConnectLocale(props.locale || "en_US");
-  const siweAuth = useSiweAuth(auth);
```

Comment:
there was a reason why I added it here and not in `inner` the reason being that otherwise we get 2 loading spinners, one while `locale` is loading and 1 while we determine auth status inside, this way we can load both at the same time and re-use the same loading state in the UI